it/controllers/benefits/TravelOrEntertainmentControllerISpec.scala

```diff
@@ -0,0 +1,492 @@
+    val expectedButtonText: String
+    val yesText: String
+    val noText: String
+    val thisIncludes: String
+  }
+
+  object ExpectedIndividualEN extends SpecificExpectedResults {
+    val expectedTitle = "Did you get any travel or entertainment benefits from this company?"
+    val expectedH1 = "Did you get any travel or entertainment benefits from this company?"
+    val expectedErrorTitle = s"Error: $expectedTitle"
+    val expectedError = "Select yes if you got travel or entertainment benefits"
+  }
+
+  object ExpectedIndividualCY extends SpecificExpectedResults {
+    val expectedTitle = "Did you get any travel or entertainment benefits from this company?"
+    val expectedH1 = "Did you get any travel or entertainment benefits from this company?"
+    val expectedErrorTitle = s"Error: $expectedTitle"
+    val expectedError = "Select yes if you got travel or entertainment benefits"
+  }
+
+  object ExpectedAgentEN extends SpecificExpectedResults {
+    val expectedTitle = "Did your client get any travel or entertainment benefits from this company?"
+    val expectedH1 = "Did your client get any travel or entertainment benefits from this company?"
+    val expectedErrorTitle = s"Error: $expectedTitle"
+    val expectedError = "Select yes if your client got travel or entertainment benefits"
+  }
+
+  object ExpectedAgentCY extends SpecificExpectedResults {
+    val expectedTitle = "Did your client get any travel or entertainment benefits from this company?"
+    val expectedH1 = "Did your client get any travel or entertainment benefits from this company?"
+    val expectedErrorTitle = s"Error: $expectedTitle"
+    val expectedError = "Select yes if your client got travel or entertainment benefits"
+  }
+
+  object CommonExpectedEN extends CommonExpectedResults {
+    val expectedCaption = s"Employment for 6 April ${taxYearEOY - 1} to 5 April $taxYearEOY"
+    val expectedButtonText = "Continue"
+    val yesText = "Yes"
+    val noText = "No"
+    val thisIncludes = "This includes benefits such as travel, incidental overnight expenses and entertaining."
+  }
+
+  object CommonExpectedCY extends CommonExpectedResults {
+    val expectedCaption = s"Employment for 6 April ${taxYearEOY - 1} to 5 April $taxYearEOY"
+    val expectedButtonText = "Continue"
+    val yesText = "Yes"
+    val noText = "No"
+    val thisIncludes = "This includes benefits such as travel, incidental overnight expenses and entertaining."
+  }
+
+  val userScenarios: Seq[UserScenario[CommonExpectedResults, SpecificExpectedResults]] = {
+    Seq(UserScenario(isWelsh = false, isAgent = false, CommonExpectedEN, Some(ExpectedIndividualEN)),
+      UserScenario(isWelsh = false, isAgent = true, CommonExpectedEN, Some(ExpectedAgentEN)),
+      UserScenario(isWelsh = true, isAgent = false, CommonExpectedCY, Some(ExpectedIndividualCY)),
+      UserScenario(isWelsh = true, isAgent = true, CommonExpectedCY, Some(ExpectedAgentCY)))
+  }
+
+  object CyaModel {
```

Comment:
this CyaModel can be removed as it's not used

Reply:
removed